src/main/java/io/jenkins/plugins/sshbuildagents/ssh/Connection.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016, Michael Clarke
```

Comment:
Not sure why you use this copyright here? 
It looks to be a new class written by you. Either you put yourself or nobody but not someone who was not part of writing this code.

Reply:
C&P from the old class, so I keep the same header

Reply:
I will review it, as it is likely new content, even though I started by copying it from the old file.